modules/priceFloors.md

```diff
@@ -11,6 +11,7 @@ pbjs.setConfig({
       enforceJS: true //defaults to true
     },
     auctionDelay: 150, // in milliseconds defaults to 0
+    floorProvider: '', //defaults to en empty string
```

Comment:
Technically no default now, just put in some random example please!

`awesomeFloorProviderName` maybe 😉

Reply:
I knew I forgot something. would you like me to note that it's an optional field?